src/basis/basis.js

```diff
@@ -396,14 +397,23 @@ export class Basis {
      */
     get maxPairwiseDistance() {
         let maxDistance = 0;
-        for (let i = 0; i < this._elements.array.length; i++) {
-            for (let j = i + 1; j < this._elements.array.length; j++) {
-                const distance = math.vDist(
-                    this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
-                if (distance > maxDistance) {
-                    maxDistance = distance;
+        if (this._elements.array.length >= 2) {
+            for (let i = 0; i < this._elements.array.length; i++) {
+                for (let j = i + 1; j < this._elements.array.length; j++) {
+                    const distance = math.vDist(
+                        this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
+                    if (distance > maxDistance) {
+                        maxDistance = distance;
+                    }
                 }
             }
+        } else if(this._elements.array.length === 1) {
+            const element = this._elements.getArrayElementByIndex(0);
+            Object.keys(PERIODIC_TABLE).forEach(key => {
+                if (element === PERIODIC_TABLE[key].symbol) {
+                    maxDistance = math.precise(PERIODIC_TABLE[key].atomic_radius_pm * 0.01, 4);
+                }
+            });
         }
         if (maxDistance === 0) {
```

Comment:
We should be able to remove this if statement, no structures with `elements.array.length <= 0`